spec/models/user_spec.rb

```diff
@@ -11,7 +11,7 @@
 #  salt               :string(255)
 #
 
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.